### ISSUE TITLE
TestServ is missing a method to add a property

### ISSUE THREAD

**Issue by HipsterZipster** [dakolech/angular-unit-testing-helpers#4]
Currently if we want to add invidual properties onto a mock service that we've generated using MockServ we cannot use the chaining pattern and need to declare the property on the object with full syntax breaking the flow.

Instead there should be an addProperty method on TestServ that although will not add a spy (since it's merely a value), it will allow us to follow a standard chaining syntax.

I plan to submit a PR within the next week for this

```
var MockServ = new TestServ();
MockServ
    .addMethod('foo', 'barVal');
    .addMethod('fooFunc', function() { return 'barFunc' } )
    .addProperty('fooStream', new Rx.Subject())
```

**Comment by dakolech** [comment]
Good idea, thanks. I will try to add this method today.

**Comment by dakolech** [comment]
It's available in release 0.0.11 now.
